### PR TITLE
Migrate tfmini driver class member variable initialization to declarations, format whitespace and alphabetize/group/order var/method declarations

### PR DESCRIPTION
**Pull request by mcsauder**
**Describe problem solved by the proposed pull request**
This PR migrates tfmini class member variable initialization to respective declarations, formats whitespace, and alphabetizes/organizes variable and method declarations in tfmini.cpp.

This PR is primarily copy/paste of the variable initialization values and method declarations along with whitespace formatting but also includes some minor standardization of the logic in `tfmini_main()` and `cycle_trampoline` variable names.

**Describe your preferred solution**
Standardizing all of the distance sensor driver variable initialization, method ordering and general style will allow for future inheritance structure work to be performed on the distance sensor driver classes.

**Describe possible alternatives**
All of the distance sensor driver work could be accomplished in one massive PR, but breaking up work in each driver should minimize risk and reduce review effort.

**Additional context**
See #9279, #11853, #11857, #1858, #11859, #11891, #11892

Please let me know if you have any questions on this PR. Thanks!

-Mark

### CODE REVIEW

**Comment by mcsauder** [comment]
@ChristophTobler , would you be willing to review this PR for me?  I am working to align the distance sensor drivers so that an inheritance structure can be created and code duplication can be reduced.  Thanks in advance for any feedback you might have!

File: src/drivers/distance_sensor/tfmini/tfmini.cpp [hunk 1]
```diff
@@ -875,52 +862,44 @@ tfmini_main(int argc, char *argv[])
 
 		default:
 			PX4_WARN("Unknown option!");
-			return -1;
+			return -PX4_ERROR;
```
Comment (ChristophTobler) [inline src/drivers/distance_sensor/tfmini/tfmini.cpp]: remove `-`

File: src/drivers/distance_sensor/tfmini/tfmini.cpp [hunk 2]
```diff
@@ -875,52 +862,44 @@ tfmini_main(int argc, char *argv[])
...
 		}
 	}
 
 	if (myoptind >= argc) {
-		goto out_error;
+		PX4_ERR("unrecognized command");
+	        tfmini::usage();
+		return -PX4_ERROR;
```
Comment (ChristophTobler) [inline src/drivers/distance_sensor/tfmini/tfmini.cpp]: same

File: src/drivers/distance_sensor/tfmini/tfmini.cpp [hunk 3]
```diff
@@ -875,52 +862,44 @@ tfmini_main(int argc, char *argv[])
...
+	// Start/load the driver.
 	if (!strcmp(argv[myoptind], "start")) {
 		if (strcmp(device_path, "") != 0) {
 			return tfmini::start(device_path, rotation);
 
 		} else {
 			PX4_WARN("Please specify device path!");
 			tfmini::usage();
-			return -1;
+			return -PX4_ERROR;
```
Comment (ChristophTobler) [inline src/drivers/distance_sensor/tfmini/tfmini.cpp]: same

File: src/drivers/distance_sensor/tfmini/tfmini.cpp (hunk 1 above)
Comment (mcsauder) [reply src/drivers/distance_sensor/tfmini/tfmini.cpp]: Thank you for catching that!

File: src/drivers/distance_sensor/tfmini/tfmini.cpp (hunk 2 above)
Comment (mcsauder) [reply src/drivers/distance_sensor/tfmini/tfmini.cpp]: Thanks! Fixed.

File: src/drivers/distance_sensor/tfmini/tfmini.cpp (hunk 3 above)
Comment (mcsauder) [reply src/drivers/distance_sensor/tfmini/tfmini.cpp]: Copy/paste. :} Fixed. Thanks!

**Comment by mcsauder** [comment]
Thanks @thomasgubler for looking it over!  I found you in the commit history, which is why I reached out.  I'll ask the group on the dev call next week to try to find someone who can hardware test it.

Thanks again for your time!

**Comment by bys1123** [comment]
I have one, gonna test tomorrow.

**Comment by mcsauder** [comment]
Bench tested on pixhawk 4 by rotating the sensor 90 degrees to measure distance between a floor and a wall, results show behavior indistinguishable from current master:
This PR:
image


master:
image

@mhkabir , would you be willing to review this PR? Thank you!

**Comment by mcsauder** [comment]
@ChristophTobler , would you be willing to review and perhaps accept/reject this PR as well?  Thank you!

**Comment by mcsauder** [comment]
Thanks @dagar!

**Pull request merged**